mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !currentTarget.href) {
```

Comment:
Nit: use `currentTarget.href == null`. The parent `<a>`'s `href`, if it wasn't previously set, is `""`.

Obviously, this is likely a user bug, but it should at least route to the default route as a fallback (it could be useful as a "go home" route):

```js
m("a", {config: m.route}, "Go to home")
```